Vizzuality/trase
# Add CTA

## Asana

https://app.asana.com/0/1200542925644189/1200529371938611/f

## Description

Add CTA

CTA version numbers have Year-Quarter with no separators format

The cta info is on the ```cta-versions-data.jsx``` file

![image](https://user-images.githubusercontent.com/9701591/137177366-023ba22d-8e8e-43b2-897b-c34a9fce09d1.png)

## Testing instructions

Open home page. You should see the CTA. After clicking on it it shouldn't show again
